hal/include/HAL/Accelerometer.h

```diff
@@ -9,11 +9,14 @@
 
 #include "HAL/Types.h"
 
-enum HAL_AccelerometerRange : int32_t {
+/* clang-format off */
```

Comment:
C89 requires `int i; for (i = 0; ...)` instead of `for (int i = 0; ...)`, so I think it's safe to say we're using C99.